### PR TITLE
Fix many typos

### PR DESCRIPTION
**Pull request by tuanvcw**
This PR fixes many typos in various places.

### CODE REVIEW

File: trustpinning/certs.go [hunk 1]
```diff
@@ -112,7 +112,7 @@ func ValidateRoot(prevRoot *data.SignedRoot, root *data.Signed, gun data.GUN, tr
 	havePrevRoot := prevRoot != nil
 	if havePrevRoot {
 		// Retrieve all the trusted certificates from our previous root
-		// Note that we do not validate expiries here since our originally trusted root might have expired certs
+		// Note that we do not validate expires here since our originally trusted root might have expired certs
```
Comment (justincormack) [inline trustpinning/certs.go]: expiries is correct.

File: tuf/data/types.go [hunk 2]
```diff
@@ -355,7 +355,7 @@ var defaultExpiryTimes = map[RoleName]time.Duration{
 	CanonicalTimestampRole: notary.Day,
 }
 
-// SetDefaultExpiryTimes allows one to change the default expiries.
+// SetDefaultExpiryTimes allows one to change the default expires.
```
Comment (justincormack) [inline tuf/data/types.go]: expiries is correct

File: trustpinning/certs.go (hunk 1 above)
Comment (tuanvcw) [reply trustpinning/certs.go]: Thank you very much, this has been reverted.

File: tuf/data/types.go (hunk 2 above)
Comment (tuanvcw) [reply tuf/data/types.go]: Thank you, this has been reverted as well.

**Comment by tuanvcw** [comment]
Hi @justincormack,
Thanks for your reviews, if there's anything else needed to be updated, please feel free to let me know :)

**Comment by tuanvcw** [comment]
Thank you very much, @HuKeping, @endophage, @justincormack

**Pull request merged**